cloudflare-pages/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Meticulous - Trigger tests for Cloudflare Pages'
+description: 'Trigger Meticulous tests for a Cloudflare Pages deployment'
```

Comment:
naming nit: I think a name that relates to how the action works, ie  something like 'create a deployment from CF pages preview environment'. What do you think?

Reply:
Sounds good!